module-check/src/test/clojure/clojure/core/typed/test/template.clj

```diff
@@ -0,0 +1,11 @@
+(ns clojure.core.typed.test.template
+  (:require 	[clojure.core.typed :as t]
+                [clojure.template :as template] 
+		[clojure.xml :as xml]
+		[clojure.zip :as zip]
+		[clojure.test :refer :all]                
+		[clojure.core.typed.test.test-utils :refer :all]))
+
+(deftest check-test
+(is-tc-e (clojure.template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5))t/Any) 
+(is-tc-err (clojure.template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5)) t/Any))
```

Comment:
I think you don't fully understand the difference between `is-tc-e` and `is-tc-err`. Like what are you supposed to test in `is-tc-e` and in `is-tc-err`? Please check this [`doseq-test`](https://github.com/clojure/core.typed/blob/master/module-check/src/test/clojure/clojure/core/typed/test/core.clj#L1298) and try to explain the difference between 2 forms and why `is-tc-e` used for the first one and `is-tc-err` for the second.